ray_rllib/basic.py

```diff
@@ -1,52 +1,58 @@
 import diambra.arena
+import gymnasium as gym
 from diambra.arena.ray_rllib.make_ray_env import DiambraArena, preprocess_ray_config
-from ray.rllib.algorithms.ppo import PPO
+from ray.rllib.algorithms.ppo import PPO, PPOConfig
+from ray.tune.logger import pretty_print
 
 def main():
-    # Settings
-    settings = {}
-    settings["hardcore"] = True
-    settings["frame_shape"] = (84, 84, 1)
+    # Environment Settings
+    env_settings = {}
+    env_settings["frame_shape"] = (84, 84, 1)
+    env_settings["action_space"] = "discrete"
```

Comment:
If we exposed a const in arena for the available action spaces, we can also use it here and prevent e.g typos

Reply:
I see the point, will look into it

Reply:
Done in arena: https://github.com/diambra/arena/commit/0770bff1e963cc9baecf06833a4555388c35557c
Will update also here